### DISCUSSION TITLE
ETA Medusa 2.0

### DISCUSSION THREAD

**Discussion by fxmb** [medusajs/medusa#6689] [Feature Requests]
Is there any update on the release of Medusa 2.0? We are looking forward to this for some time now :-) Thanks for the great work

**Comment by olivermrbl** [comment]
We don't have a set release date yet, but we expect a release candidate to be announced in April and the official release some time after summer.

**Comment by boringcommerce** [comment]
Any tentative date for that?

**Reply by boringcommerce** [reply]
And how easy will it be to upgrade our existing medusa instance to 2.0?

**Reply by olivermrbl** [reply]
We still can't give you a specific date for the release. We are finalizing an early preview version that will go out in a few weeks.

Regarding the upgrade, we want to make the migration as smooth as possible. However, there will be breaking changes, such as in the Rest API and around customization, so some effort will be required.

**Reply by boringcommerce** [reply]
Okay, that's not bad at all. Thanks man.

**Comment by SalahAdDin** [comment]
Does it include multi-language support?

**Comment by Deroswent** [comment]
Guys, roll out the early version. This is what we need to start new projects.  Thanks for the great work  :+1:

**Comment by MrCasquette** [comment]
@olivermrbl  You should definitely release a first version so that we can adapt to the new principles and share feedback on UX and bugs :) 
I'm now stuck on my new project. I can't start a V1 because it will be deprecated, but I can't start a V2 either... :(

**Reply by Noiztbander** [reply]
I'm the same :(  i have been working this 3 past months in a ecommerce but now im stuck waiting for the v2 version to migrate it. meanwhile im more focused on upgrading the product but still here, waiting

**Reply by olivermrbl** [reply]
Hey guys, I missed these message, apologise. Have you given the preview (now RC) version a spin?

**Comment by muthukub** [comment]
@olivermrbl  - 2.0 preview is very good starting point. Thank you for all of your hardworking guys. when can we expect 2.0 stable release? is there any tentative release data you guys have in mind?

**Reply by olivermrbl** [reply]
Official release is expected October 23rd

**Comment by Jonneal3** [comment]
What in the new one?? @olivermrbl

**Comment by luluhoc** [comment]
I guess they abandoned all the v1 users, there is no migration guide, and the v1 release was abandoned, left with all the errors.

**Reply by SalahAdDin** [reply]
> I guess they abandoned all the v1 users, there is no migration guide, and the v1 release was abandoned left with all the errors.

It is more like the are busy delivering stable version 2 before moving resources to handle version 1. Once v2 is stable and satisfactory, they will continue working on v1 also.

**Reply by luluhoc** [reply]
I don't think so, because they closed all the PR for v1 that people worked on which featured bug fixes. I guess profits before users. 

> Medusa Cloud

**Reply by SalahAdDin** [reply]
> I don't think so, because they closed all the PR for v1 that people worked on which featured bug fixes. I guess profits before users.
> 
> > Medusa Cloud

any alternative?

**Reply by alessioacella** [reply]
Any news on this?
